css/styless.css

```diff
@@ -597,3 +606,258 @@ li {
+    justify-content: space-between;
+    align-items: flex-end;
+    padding: 26px;
+    margin: 0;
+  }
+
+  .logo {
+    display: flex;
+    flex-direction: row;
+    margin-left: 90px;
+    margin-bottom: 10px;
+  }
+
+  .headline-description {
+    width: 566px;
+    height: 123px;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .email {
+    align-self: center;
+    margin-right: 20px;
+  }
+
+  .desktop-headline {
+    padding-left: 250px;
+    padding-bottom: 100px;
+    margin: 0;
+    font-family: "Crete Round", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 65px;
+    line-height: 64px;
+    letter-spacing: 0.37px;
+    color: #f55800;
+    order: 0;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    background-position: right;
+    padding: 120px 0 300px 0;
+    margin: 0;
+    background-image: url(../img/illustraction/Illustration-header.svg);
+    background-repeat: no-repeat;
+  }
+
+  .social-media {
+    position: absolute;
+    height: auto;
+    list-style-type: none;
+    flex-direction: column;
+    flex-basis: 0;
+    flex-shrink: 0;
+  }
+
+  .works {
+    padding: 100px 100px 80px 100px;
+    margin: 0;
+  }
+
+  .works-header-row {
+    display: flex;
+    flex-direction: row;
+    width: 900px;
+  }
+
+  .h2-vector {
+    margin-top: 78px;
+    width: 60%;
+  }
+
+  .featured-image {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+  }
+
+  .placeholder-img {
+    width: 684px;
+    height: 448px;
+  }
+
+  .right-block {
+    margin-top: 100px;
+  }
+
+  .right-block-btn {
+    display: block;
+  }
+
+  .word-cards {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .work {
+    margin-left: 30px;
+  }
+
+  .work1 {
+    padding-top: 0;
+  }
+
+  .work-card1 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin-right: 30px;
+  }
+
+  .about-me-headline-container {
+    background-image: url(../img/illustraction/illustration-about\ me.svg);
+    background-repeat: no-repeat;
+    background-position: right;
+    padding: 120px 0 120px 0;
+    width: 100%;
+  }
+
+  .about-me-headline {
+    display: flex;
+    flex-direction: column;
+    padding: 0 0 0 200px;
+    width: 30%;
+  }
+
+  .about-me-header {
+    width: 566px;
+    height: 78px;
+  }
+
+  .about-me-description {
+    width: 566px;
+    height: 148px;
+  }
+
+  .resourcesection {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    gap: 152px;
+    margin: auto;
+  }
+
+  .source {
+    display: flex;
+    flex-direction: column;
+    margin: auto;
+  }
+
+  .subline {
+    margin-right: auto;
+    text-align: left;
+  }
+
+  .form-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    width: auto;
+  }
+
+  .form-header {
+    width: 455px;
+    height: 256px;
+    color: #262626;
+    padding-left: 300px;
+  }
```

Comment:
- [OPTIONAL] You can try and remove the padding-left from this element, and adding some margin and width to the form (.form class) to center the items in the contact section.

before the edit:
![image](https://user-images.githubusercontent.com/99830261/193172092-a28cb0c1-1509-423c-a14f-43a165af9470.png)

after the edit:
![image](https://user-images.githubusercontent.com/99830261/193171411-085ba8cd-b353-454e-a708-a56b22326d9c.png)